.markdownlintignore

```diff
@@ -1 +1,2 @@
 node_modules/
+docs/home.md
```

Comment:
doesnt that work with a at the beginning of the file?
```html
<!-- markdownlint-disable MD041 MD033 -->
```

Reply:
yeah this works